host/sgx/sgxquote.c

```diff
@@ -135,6 +136,191 @@ static quote3_error_t (*_sgx_qv_verify_quote)(
 static void* _ql_module;
 static void* _qvl_module;
 
+// This is a helper for getting "human readable error messages" - as mentioned
+// in issue 2945 - so that we do not have to return complex quote3_error_t codes
+// in error messages.
+static const char* get_quote3_error_t_string(quote3_error_t err)
```

Comment:
```suggestion
static const char* get_quote3_error_t_string(quote3_error_t error)
```
avoid abbreviations, per OE style guidelines